interfaces/seccomp/template.go

```diff
@@ -161,9 +161,9 @@ inotify_init
 inotify_init1
 inotify_rm_watch
 
-# TIOCSTI allows for faking input (man tty_ioctl)
-# TODO: this should be scaled back even more
-ioctl - !TIOCSTI
+# FIXME: replace this with the filter of TIOCSTI once snap-confine can read this syntax
+# See LP:#1662489 for context.
+ioctl
```

Comment:
We've discussed this on a hangout so I'll just reply with a quick summary:
 - this is a temporary fix to unbreak the world
 - we will use snap-confine from the core
 - the patch a09abe79fb0acb335b681ae2f9a7f106c6ad1490 should be reverted as a part of that work